### PR TITLE
fix: resolve generate_block_with_template rpc error

### PR DESCRIPTION
**Pull request by quake**
fix a bug which is introduced by https://github.com/nervosnetwork/ckb/pull/2569: this rpc will fail after height 11 on a local dev chain. This PR changed an integration test also to utilize this rpc to make issue it's covered by testing.

### CODE REVIEW

**Comment by nervos-bot-user** [comment]
Can one of the admins verify this patch?

**Review by doitian — APPROVED** [review]

**Review by zhangsoledad — APPROVED** [review]

**Comment by quake** [comment]
bors r=doitian,zhangsoledad

File: rpc/src/module/test.rs [hunk 1]
```diff
@@ -170,13 +170,10 @@ impl IntegrationTestRpc for IntegrationTestRpcImpl {
...
-            .transactions
-            .iter()
-            .map(|tx_template| {
-                let transaction: packed::Transaction = tx_template.data.clone().into();
-                transaction.into_view()
-            })
+        let txs: Vec<_> = packed::Block::from(block_template.clone())
+            .transactions()
+            .into_iter()
+            .map(|tx| tx.into_view())
```
Comment (keroro520) [inline rpc/src/module/test.rs:176]: What bug is this PR supposed to fix?

Comment (doitian) [reply rpc/src/module/test.rs:176]: Cellbase is not included in `block_template.transactions`.

**Review by keroro520 — APPROVED** [review]

**Pull request merged**